core/src/components/modal/modal.tsx

```diff
@@ -761,7 +773,55 @@ export class Modal implements ComponentInterface, OverlayInterface {
     return this.currentBreakpoint;
   }
 
+  private async moveToNextBreakpoint() {
+    const { breakpoints, currentBreakpoint } = this;
+
+    if (!breakpoints || currentBreakpoint == null) {
+      /**
+       * If the modal does not have breakpoints and/or the current
+       * breakpoint is not set, we can't move to the next breakpoint.
+       */
+      return false;
```

Comment:
We don't seem to use the return value anywhere. Was this added in case we do need the results in the future?

Reply:
Correct. The intention was that it allows us to make decisions in the future based on if the operation was successful or not. I could be convinced that this is not necessary since the method is not part of a public API that developers can rely on.

Reply:
probably fine to keep it 👍